modules/search/src/model.rs

```diff
@@ -19,6 +19,10 @@ pub struct SearchOptions {
 pub struct FoundAdvisory {
```

Comment:
I've similar DTOs (AdvisorySummary, AdvisoryDetails) in the graph models for DTOing.